pilot/pkg/networking/grpcgen/grpcgen.go

```diff
@@ -0,0 +1,230 @@
+// Copyright 2017 Istio Authors
+//
```

Comment:
what is the difference between grpcgen and apigen?

Reply:
Grpcgen supports 'native'/proxy-less GRPC.

Apigen supports same features that MCP provided in 1.5 - without the complexity. And is a step in the optimization of the DNS (and other things)